// mysql3/SystemVariables.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var SessionVariable;

(function (SessionVariable) {
  SessionVariable["AriaBlockSize"] = "aria_block_size";
  SessionVariable["AriaCheckpointInterval"] = "aria_checkpoint_interval";
  SessionVariable["AriaCheckpointLogActivity"] = "aria_checkpoint_log_activity";
  SessionVariable["AriaEncryptTables"] = "aria_encrypt_tables";
  SessionVariable["AriaForceStartAfterRecoveryFailures"] = "aria_force_start_after_recovery_failures";
  SessionVariable["AriaGroupCommit"] = "aria_group_commit";
  SessionVariable["AriaGroupCommitInterval"] = "aria_group_commit_interval";
  SessionVariable["AriaLogFileSize"] = "aria_log_file_size";
  SessionVariable["AriaLogPurgeType"] = "aria_log_purge_type";
  SessionVariable["AriaMaxSortFileSize"] = "aria_max_sort_file_size";
  SessionVariable["AriaPageChecksum"] = "aria_page_checksum";
  SessionVariable["AriaPagecacheAgeThreshold"] = "aria_pagecache_age_threshold";
  SessionVariable["AriaPagecacheBufferSize"] = "aria_pagecache_buffer_size";
  SessionVariable["AriaPagecacheDivisionLimit"] = "aria_pagecache_division_limit";
  SessionVariable["AriaPagecacheFileHashSize"] = "aria_pagecache_file_hash_size";
  SessionVariable["AriaRecover"] = "aria_recover";
  SessionVariable["AriaRepairThreads"] = "aria_repair_threads";
  SessionVariable["AriaSortBufferSize"] = "aria_sort_buffer_size";
  SessionVariable["AriaStatsMethod"] = "aria_stats_method";
  SessionVariable["AriaSyncLogDir"] = "aria_sync_log_dir";
  SessionVariable["AriaUsedForTempTables"] = "aria_used_for_temp_tables";
  SessionVariable["AutoIncrementIncrement"] = "auto_increment_increment";
  SessionVariable["AutoIncrementOffset"] = "auto_increment_offset";
  SessionVariable["Autocommit"] = "autocommit";
  SessionVariable["AutomaticSpPrivileges"] = "automatic_sp_privileges";
  SessionVariable["BackLog"] = "back_log";
  SessionVariable["Basedir"] = "basedir";
  SessionVariable["BigTables"] = "big_tables";
  SessionVariable["BinlogAnnotateRowEvents"] = "binlog_annotate_row_events";
  SessionVariable["BinlogCacheSize"] = "binlog_cache_size";
  SessionVariable["BinlogChecksum"] = "binlog_checksum";
  SessionVariable["BinlogCommitWaitCount"] = "binlog_commit_wait_count";
  SessionVariable["BinlogCommitWaitUsec"] = "binlog_commit_wait_usec";
  SessionVariable["BinlogDirectNonTransactionalUpdates"] = "binlog_direct_non_transactional_updates";
  SessionVariable["BinlogFormat"] = "binlog_format";
  SessionVariable["BinlogOptimizeThreadScheduling"] = "binlog_optimize_thread_scheduling";
  SessionVariable["BinlogRowImage"] = "binlog_row_image";
  SessionVariable["BinlogStmtCacheSize"] = "binlog_stmt_cache_size";
  SessionVariable["BulkInsertBufferSize"] = "bulk_insert_buffer_size";
  SessionVariable["CharacterSetClient"] = "character_set_client";
  SessionVariable["CharacterSetConnection"] = "character_set_connection";
  SessionVariable["CharacterSetDatabase"] = "character_set_database";
  SessionVariable["CharacterSetFilesystem"] = "character_set_filesystem";
  SessionVariable["CharacterSetResults"] = "character_set_results";
  SessionVariable["CharacterSetServer"] = "character_set_server";
  SessionVariable["CharacterSetSystem"] = "character_set_system";
  SessionVariable["CharacterSetsDir"] = "character_sets_dir";
  SessionVariable["CollationConnection"] = "collation_connection";
  SessionVariable["CollationDatabase"] = "collation_database";
  SessionVariable["CollationServer"] = "collation_server";
  SessionVariable["CompletionType"] = "completion_type";
  SessionVariable["ConcurrentInsert"] = "concurrent_insert";
  SessionVariable["ConnectTimeout"] = "connect_timeout";
  SessionVariable["Datadir"] = "datadir";
  SessionVariable["DateFormat"] = "date_format";
  SessionVariable["DatetimeFormat"] = "datetime_format";
  SessionVariable["DeadlockSearchDepthLong"] = "deadlock_search_depth_long";
  SessionVariable["DeadlockSearchDepthShort"] = "deadlock_search_depth_short";
  SessionVariable["DeadlockTimeoutLong"] = "deadlock_timeout_long";
  SessionVariable["DeadlockTimeoutShort"] = "deadlock_timeout_short";
  SessionVariable["DebugNoThreadAlarm"] = "debug_no_thread_alarm";
  SessionVariable["DefaultMasterConnection"] = "default_master_connection";
  SessionVariable["DefaultRegexFlags"] = "default_regex_flags";
  SessionVariable["DefaultStorageEngine"] = "default_storage_engine";
  SessionVariable["DefaultTmpStorageEngine"] = "default_tmp_storage_engine";
  SessionVariable["DefaultWeekFormat"] = "default_week_format";
  SessionVariable["DelayKeyWrite"] = "delay_key_write";
  SessionVariable["DelayedInsertLimit"] = "delayed_insert_limit";
  SessionVariable["DelayedInsertTimeout"] = "delayed_insert_timeout";
  SessionVariable["DelayedQueueSize"] = "delayed_queue_size";
  SessionVariable["DivPrecisionIncrement"] = "div_precision_increment";
  SessionVariable["EncryptBinlog"] = "encrypt_binlog";
  SessionVariable["EncryptTmpDiskTables"] = "encrypt_tmp_disk_tables";
  SessionVariable["EncryptTmpFiles"] = "encrypt_tmp_files";
  SessionVariable["EnforceStorageEngine"] = "enforce_storage_engine";
  SessionVariable["ErrorCount"] = "error_count";
  SessionVariable["EventScheduler"] = "event_scheduler";
  SessionVariable["ExpensiveSubqueryLimit"] = "expensive_subquery_limit";
  SessionVariable["ExpireLogsDays"] = "expire_logs_days";
  SessionVariable["ExplicitDefaultsForTimestamp"] = "explicit_defaults_for_timestamp";
  SessionVariable["ExternalUser"] = "external_user";
  SessionVariable["ExtraMaxConnections"] = "extra_max_connections";
  SessionVariable["ExtraPort"] = "extra_port";
  SessionVariable["Flush"] = "flush";
  SessionVariable["FlushTime"] = "flush_time";
  SessionVariable["ForeignKeyChecks"] = "foreign_key_checks";
  SessionVariable["FtBooleanSyntax"] = "ft_boolean_syntax";
  SessionVariable["FtMaxWordLen"] = "ft_max_word_len";
  SessionVariable["FtMinWordLen"] = "ft_min_word_len";
  SessionVariable["FtQueryExpansionLimit"] = "ft_query_expansion_limit";
  SessionVariable["FtStopwordFile"] = "ft_stopword_file";
  SessionVariable["GeneralLog"] = "general_log";
  SessionVariable["GeneralLogFile"] = "general_log_file";
  SessionVariable["GroupConcatMaxLen"] = "group_concat_max_len";
  SessionVariable["GtidBinlogPos"] = "gtid_binlog_pos";
  SessionVariable["GtidBinlogState"] = "gtid_binlog_state";
  SessionVariable["GtidCurrentPos"] = "gtid_current_pos";
  SessionVariable["GtidDomainId"] = "gtid_domain_id";
  SessionVariable["GtidIgnoreDuplicates"] = "gtid_ignore_duplicates";
  SessionVariable["GtidSeqNo"] = "gtid_seq_no";
  SessionVariable["GtidSlavePos"] = "gtid_slave_pos";
  SessionVariable["GtidStrictMode"] = "gtid_strict_mode";
  SessionVariable["HaveCompress"] = "have_compress";
  SessionVariable["HaveCrypt"] = "have_crypt";
  SessionVariable["HaveDynamicLoading"] = "have_dynamic_loading";
  SessionVariable["HaveGeometry"] = "have_geometry";
  SessionVariable["HaveOpenssl"] = "have_openssl";
  SessionVariable["HaveProfiling"] = "have_profiling";
  SessionVariable["HaveQueryCache"] = "have_query_cache";
  SessionVariable["HaveRtreeKeys"] = "have_rtree_keys";
  SessionVariable["HaveSsl"] = "have_ssl";
  SessionVariable["HaveSymlink"] = "have_symlink";
  SessionVariable["HistogramSize"] = "histogram_size";
  SessionVariable["HistogramType"] = "histogram_type";
  SessionVariable["HostCacheSize"] = "host_cache_size";
  SessionVariable["Hostname"] = "hostname";
  SessionVariable["Identity"] = "identity";
  SessionVariable["IgnoreBuiltinInnodb"] = "ignore_builtin_innodb";
  SessionVariable["IgnoreDbDirs"] = "ignore_db_dirs";
  SessionVariable["InTransaction"] = "in_transaction";
  SessionVariable["InitConnect"] = "init_connect";
  SessionVariable["InitFile"] = "init_file";
  SessionVariable["InitSlave"] = "init_slave";
  SessionVariable["InnodbAdaptiveFlushing"] = "innodb_adaptive_flushing";
  SessionVariable["InnodbAdaptiveFlushingLwm"] = "innodb_adaptive_flushing_lwm";
  SessionVariable["InnodbAdaptiveHashIndex"] = "innodb_adaptive_hash_index";
  SessionVariable["InnodbAdaptiveHashIndexPartitions"] = "innodb_adaptive_hash_index_partitions";
  SessionVariable["InnodbAdaptiveMaxSleepDelay"] = "innodb_adaptive_max_sleep_delay";
  SessionVariable["InnodbAdditionalMemPoolSize"] = "innodb_additional_mem_pool_size";
  SessionVariable["InnodbApiBkCommitInterval"] = "innodb_api_bk_commit_interval";
  SessionVariable["InnodbApiDisableRowlock"] = "innodb_api_disable_rowlock";
  SessionVariable["InnodbApiEnableBinlog"] = "innodb_api_enable_binlog";
  SessionVariable["InnodbApiEnableMdl"] = "innodb_api_enable_mdl";
  SessionVariable["InnodbApiTrxLevel"] = "innodb_api_trx_level";
  SessionVariable["InnodbAutoextendIncrement"] = "innodb_autoextend_increment";
  SessionVariable["InnodbAutoincLockMode"] = "innodb_autoinc_lock_mode";
  SessionVariable["InnodbBackgroundScrubDataCheckInterval"] = "innodb_background_scrub_data_check_interval";
  SessionVariable["InnodbBackgroundScrubDataCompressed"] = "innodb_background_scrub_data_compressed";
  SessionVariable["InnodbBackgroundScrubDataInterval"] = "innodb_background_scrub_data_interval";
  SessionVariable["InnodbBackgroundScrubDataUncompressed"] = "innodb_background_scrub_data_uncompressed";
  SessionVariable["InnodbBufDumpStatusFrequency"] = "innodb_buf_dump_status_frequency";
  SessionVariable["InnodbBufferPoolDumpAtShutdown"] = "innodb_buffer_pool_dump_at_shutdown";
  SessionVariable["InnodbBufferPoolDumpNow"] = "innodb_buffer_pool_dump_now";
  SessionVariable["InnodbBufferPoolDumpPct"] = "innodb_buffer_pool_dump_pct";
  SessionVariable["InnodbBufferPoolFilename"] = "innodb_buffer_pool_filename";
  SessionVariable["InnodbBufferPoolInstances"] = "innodb_buffer_pool_instances";
  SessionVariable["InnodbBufferPoolLoadAbort"] = "innodb_buffer_pool_load_abort";
  SessionVariable["InnodbBufferPoolLoadAtStartup"] = "innodb_buffer_pool_load_at_startup";
  SessionVariable["InnodbBufferPoolLoadNow"] = "innodb_buffer_pool_load_now";
  SessionVariable["InnodbBufferPoolPopulate"] = "innodb_buffer_pool_populate";
  SessionVariable["InnodbBufferPoolSize"] = "innodb_buffer_pool_size";
  SessionVariable["InnodbChangeBufferMaxSize"] = "innodb_change_buffer_max_size";
  SessionVariable["InnodbChangeBuffering"] = "innodb_change_buffering";
  SessionVariable["InnodbChecksumAlgorithm"] = "innodb_checksum_algorithm";
  SessionVariable["InnodbChecksums"] = "innodb_checksums";
  SessionVariable["InnodbCleanerLsnAgeFactor"] = "innodb_cleaner_lsn_age_factor";
  SessionVariable["InnodbCmpPerIndexEnabled"] = "innodb_cmp_per_index_enabled";
  SessionVariable["InnodbCommitConcurrency"] = "innodb_commit_concurrency";
  SessionVariable["InnodbCompressionAlgorithm"] = "innodb_compression_algorithm";
  SessionVariable["InnodbCompressionFailureThresholdPct"] = "innodb_compression_failure_threshold_pct";
  SessionVariable["InnodbCompressionLevel"] = "innodb_compression_level";
  SessionVariable["InnodbCompressionPadPctMax"] = "innodb_compression_pad_pct_max";
  SessionVariable["InnodbConcurrencyTickets"] = "innodb_concurrency_tickets";
  SessionVariable["InnodbCorruptTableAction"] = "innodb_corrupt_table_action";
  SessionVariable["InnodbDataFilePath"] = "innodb_data_file_path";
  SessionVariable["InnodbDataHomeDir"] = "innodb_data_home_dir";
  SessionVariable["InnodbDefaultEncryptionKeyId"] = "innodb_default_encryption_key_id";
  SessionVariable["InnodbDefragment"] = "innodb_defragment";
  SessionVariable["InnodbDefragmentFillFactor"] = "innodb_defragment_fill_factor";
  SessionVariable["InnodbDefragmentFillFactorNRecs"] = "innodb_defragment_fill_factor_n_recs";
  SessionVariable["InnodbDefragmentFrequency"] = "innodb_defragment_frequency";
  SessionVariable["InnodbDefragmentNPages"] = "innodb_defragment_n_pages";
  SessionVariable["InnodbDefragmentStatsAccuracy"] = "innodb_defragment_stats_accuracy";
  SessionVariable["InnodbDisableSortFileCache"] = "innodb_disable_sort_file_cache";
  SessionVariable["InnodbDisallowWrites"] = "innodb_disallow_writes";
  SessionVariable["InnodbDoublewrite"] = "innodb_doublewrite";
  SessionVariable["InnodbEmptyFreeListAlgorithm"] = "innodb_empty_free_list_algorithm";
  SessionVariable["InnodbEncryptLog"] = "innodb_encrypt_log";
  SessionVariable["InnodbEncryptTables"] = "innodb_encrypt_tables";
  SessionVariable["InnodbEncryptionRotateKeyAge"] = "innodb_encryption_rotate_key_age";
  SessionVariable["InnodbEncryptionRotationIops"] = "innodb_encryption_rotation_iops";
  SessionVariable["InnodbEncryptionThreads"] = "innodb_encryption_threads";
  SessionVariable["InnodbFakeChanges"] = "innodb_fake_changes";
  SessionVariable["InnodbFastShutdown"] = "innodb_fast_shutdown";
  SessionVariable["InnodbFatalSemaphoreWaitThreshold"] = "innodb_fatal_semaphore_wait_threshold";
  SessionVariable["InnodbFileFormat"] = "innodb_file_format";
  SessionVariable["InnodbFileFormatCheck"] = "innodb_file_format_check";
  SessionVariable["InnodbFileFormatMax"] = "innodb_file_format_max";
  SessionVariable["InnodbFilePerTable"] = "innodb_file_per_table";
  SessionVariable["InnodbFlushLogAtTimeout"] = "innodb_flush_log_at_timeout";
  SessionVariable["InnodbFlushLogAtTrxCommit"] = "innodb_flush_log_at_trx_commit";
  SessionVariable["InnodbFlushMethod"] = "innodb_flush_method";
  SessionVariable["InnodbFlushNeighbors"] = "innodb_flush_neighbors";
  SessionVariable["InnodbFlushingAvgLoops"] = "innodb_flushing_avg_loops";
  SessionVariable["InnodbForceLoadCorrupted"] = "innodb_force_load_corrupted";
  SessionVariable["InnodbForcePrimaryKey"] = "innodb_force_primary_key";
  SessionVariable["InnodbForceRecovery"] = "innodb_force_recovery";
  SessionVariable["InnodbForegroundPreflush"] = "innodb_foreground_preflush";
  SessionVariable["InnodbFtAuxTable"] = "innodb_ft_aux_table";
  SessionVariable["InnodbFtCacheSize"] = "innodb_ft_cache_size";
  SessionVariable["InnodbFtEnableDiagPrint"] = "innodb_ft_enable_diag_print";
  SessionVariable["InnodbFtEnableStopword"] = "innodb_ft_enable_stopword";
  SessionVariable["InnodbFtMaxTokenSize"] = "innodb_ft_max_token_size";
  SessionVariable["InnodbFtMinTokenSize"] = "innodb_ft_min_token_size";
  SessionVariable["InnodbFtNumWordOptimize"] = "innodb_ft_num_word_optimize";
  SessionVariable["InnodbFtResultCacheLimit"] = "innodb_ft_result_cache_limit";
  SessionVariable["InnodbFtServerStopwordTable"] = "innodb_ft_server_stopword_table";
  SessionVariable["InnodbFtSortPllDegree"] = "innodb_ft_sort_pll_degree";
  SessionVariable["InnodbFtTotalCacheSize"] = "innodb_ft_total_cache_size";
  SessionVariable["InnodbFtUserStopwordTable"] = "innodb_ft_user_stopword_table";
  SessionVariable["InnodbIdleFlushPct"] = "innodb_idle_flush_pct";
  SessionVariable["InnodbImmediateScrubDataUncompressed"] = "innodb_immediate_scrub_data_uncompressed";
  SessionVariable["InnodbInstrumentSemaphores"] = "innodb_instrument_semaphores";
  SessionVariable["InnodbIoCapacity"] = "innodb_io_capacity";
  SessionVariable["InnodbIoCapacityMax"] = "innodb_io_capacity_max";
  SessionVariable["InnodbKillIdleTransaction"] = "innodb_kill_idle_transaction";
  SessionVariable["InnodbLargePrefix"] = "innodb_large_prefix";
  SessionVariable["InnodbLockWaitTimeout"] = "innodb_lock_wait_timeout";
  SessionVariable["InnodbLockingFakeChanges"] = "innodb_locking_fake_changes";
  SessionVariable["InnodbLocksUnsafeForBinlog"] = "innodb_locks_unsafe_for_binlog";
  SessionVariable["InnodbLogArchDir"] = "innodb_log_arch_dir";
  SessionVariable["InnodbLogArchExpireSec"] = "innodb_log_arch_expire_sec";
  SessionVariable["InnodbLogArchive"] = "innodb_log_archive";
  SessionVariable["InnodbLogBlockSize"] = "innodb_log_block_size";
  SessionVariable["InnodbLogBufferSize"] = "innodb_log_buffer_size";
  SessionVariable["InnodbLogChecksumAlgorithm"] = "innodb_log_checksum_algorithm";
  SessionVariable["InnodbLogCompressedPages"] = "innodb_log_compressed_pages";
  SessionVariable["InnodbLogFileSize"] = "innodb_log_file_size";
  SessionVariable["InnodbLogFilesInGroup"] = "innodb_log_files_in_group";
  SessionVariable["InnodbLogGroupHomeDir"] = "innodb_log_group_home_dir";
  SessionVariable["InnodbLruScanDepth"] = "innodb_lru_scan_depth";
  SessionVariable["InnodbMaxBitmapFileSize"] = "innodb_max_bitmap_file_size";
  SessionVariable["InnodbMaxChangedPages"] = "innodb_max_changed_pages";
  SessionVariable["InnodbMaxDirtyPagesPct"] = "innodb_max_dirty_pages_pct";
  SessionVariable["InnodbMaxDirtyPagesPctLwm"] = "innodb_max_dirty_pages_pct_lwm";
  SessionVariable["InnodbMaxPurgeLag"] = "innodb_max_purge_lag";
  SessionVariable["InnodbMaxPurgeLagDelay"] = "innodb_max_purge_lag_delay";
  SessionVariable["InnodbMirroredLogGroups"] = "innodb_mirrored_log_groups";
  SessionVariable["InnodbMonitorDisable"] = "innodb_monitor_disable";
  SessionVariable["InnodbMonitorEnable"] = "innodb_monitor_enable";
  SessionVariable["InnodbMonitorReset"] = "innodb_monitor_reset";
  SessionVariable["InnodbMonitorResetAll"] = "innodb_monitor_reset_all";
  SessionVariable["InnodbMtflushThreads"] = "innodb_mtflush_threads";
  SessionVariable["InnodbOldBlocksPct"] = "innodb_old_blocks_pct";
  SessionVariable["InnodbOldBlocksTime"] = "innodb_old_blocks_time";
  SessionVariable["InnodbOnlineAlterLogMaxSize"] = "innodb_online_alter_log_max_size";
  SessionVariable["InnodbOpenFiles"] = "innodb_open_files";
  SessionVariable["InnodbOptimizeFulltextOnly"] = "innodb_optimize_fulltext_only";
  SessionVariable["InnodbPageSize"] = "innodb_page_size";
  SessionVariable["InnodbPrefixIndexClusterOptimization"] = "innodb_prefix_index_cluster_optimization";
  SessionVariable["InnodbPrintAllDeadlocks"] = "innodb_print_all_deadlocks";
  SessionVariable["InnodbPurgeBatchSize"] = "innodb_purge_batch_size";
  SessionVariable["InnodbPurgeThreads"] = "innodb_purge_threads";
  SessionVariable["InnodbRandomReadAhead"] = "innodb_random_read_ahead";
  SessionVariable["InnodbReadAheadThreshold"] = "innodb_read_ahead_threshold";
  SessionVariable["InnodbReadIoThreads"] = "innodb_read_io_threads";
  SessionVariable["InnodbReadOnly"] = "innodb_read_only";
  SessionVariable["InnodbReplicationDelay"] = "innodb_replication_delay";
  SessionVariable["InnodbRollbackOnTimeout"] = "innodb_rollback_on_timeout";
  SessionVariable["InnodbRollbackSegments"] = "innodb_rollback_segments";
  SessionVariable["InnodbSchedPriorityCleaner"] = "innodb_sched_priority_cleaner";
  SessionVariable["InnodbScrubLog"] = "innodb_scrub_log";
  SessionVariable["InnodbScrubLogSpeed"] = "innodb_scrub_log_speed";
  SessionVariable["InnodbShowLocksHeld"] = "innodb_show_locks_held";
  SessionVariable["InnodbShowVerboseLocks"] = "innodb_show_verbose_locks";
  SessionVariable["InnodbSimulateCompFailures"] = "innodb_simulate_comp_failures";
  SessionVariable["InnodbSortBufferSize"] = "innodb_sort_buffer_size";
  SessionVariable["InnodbSpinWaitDelay"] = "innodb_spin_wait_delay";
  SessionVariable["InnodbStatsAutoRecalc"] = "innodb_stats_auto_recalc";
  SessionVariable["InnodbStatsMethod"] = "innodb_stats_method";
  SessionVariable["InnodbStatsModifiedCounter"] = "innodb_stats_modified_counter";
  SessionVariable["InnodbStatsOnMetadata"] = "innodb_stats_on_metadata";
  SessionVariable["InnodbStatsPersistent"] = "innodb_stats_persistent";
  SessionVariable["InnodbStatsPersistentSamplePages"] = "innodb_stats_persistent_sample_pages";
  SessionVariable["InnodbStatsSamplePages"] = "innodb_stats_sample_pages";
  SessionVariable["InnodbStatsTraditional"] = "innodb_stats_traditional";
  SessionVariable["InnodbStatsTransientSamplePages"] = "innodb_stats_transient_sample_pages";
  SessionVariable["InnodbStatusOutput"] = "innodb_status_output";
  SessionVariable["InnodbStatusOutputLocks"] = "innodb_status_output_locks";
  SessionVariable["InnodbStrictMode"] = "innodb_strict_mode";
  SessionVariable["InnodbSupportXa"] = "innodb_support_xa";
  SessionVariable["InnodbSyncArraySize"] = "innodb_sync_array_size";
  SessionVariable["InnodbSyncSpinLoops"] = "innodb_sync_spin_loops";
  SessionVariable["InnodbTableLocks"] = "innodb_table_locks";
  SessionVariable["InnodbThreadConcurrency"] = "innodb_thread_concurrency";
  SessionVariable["InnodbThreadSleepDelay"] = "innodb_thread_sleep_delay";
  SessionVariable["InnodbTmpdir"] = "innodb_tmpdir";
  SessionVariable["InnodbTrackChangedPages"] = "innodb_track_changed_pages";
  SessionVariable["InnodbUndoDirectory"] = "innodb_undo_directory";
  SessionVariable["InnodbUndoLogs"] = "innodb_undo_logs";
  SessionVariable["InnodbUndoTablespaces"] = "innodb_undo_tablespaces";
  SessionVariable["InnodbUseAtomicWrites"] = "innodb_use_atomic_writes";
  SessionVariable["InnodbUseFallocate"] = "innodb_use_fallocate";
  SessionVariable["InnodbUseGlobalFlushLogAtTrxCommit"] = "innodb_use_global_flush_log_at_trx_commit";
  SessionVariable["InnodbUseMtflush"] = "innodb_use_mtflush";
  SessionVariable["InnodbUseNativeAio"] = "innodb_use_native_aio";
  SessionVariable["InnodbUseStacktrace"] = "innodb_use_stacktrace";
  SessionVariable["InnodbUseSysMalloc"] = "innodb_use_sys_malloc";
  SessionVariable["InnodbUseTrim"] = "innodb_use_trim";
  SessionVariable["InnodbVersion"] = "innodb_version";
  SessionVariable["InnodbWriteIoThreads"] = "innodb_write_io_threads";
  SessionVariable["InsertId"] = "insert_id";
  SessionVariable["InteractiveTimeout"] = "interactive_timeout";
  SessionVariable["JoinBufferSize"] = "join_buffer_size";
  SessionVariable["JoinBufferSpaceLimit"] = "join_buffer_space_limit";
  SessionVariable["JoinCacheLevel"] = "join_cache_level";
  SessionVariable["KeepFilesOnCreate"] = "keep_files_on_create";
  SessionVariable["KeyBufferSize"] = "key_buffer_size";
  SessionVariable["KeyCacheAgeThreshold"] = "key_cache_age_threshold";
  SessionVariable["KeyCacheBlockSize"] = "key_cache_block_size";
  SessionVariable["KeyCacheDivisionLimit"] = "key_cache_division_limit";
  SessionVariable["KeyCacheFileHashSize"] = "key_cache_file_hash_size";
  SessionVariable["KeyCacheSegments"] = "key_cache_segments";
  SessionVariable["LargeFilesSupport"] = "large_files_support";
  SessionVariable["LargePageSize"] = "large_page_size";
  SessionVariable["LargePages"] = "large_pages";
  SessionVariable["LastGtid"] = "last_gtid";
  SessionVariable["LastInsertId"] = "last_insert_id";
  SessionVariable["LcMessages"] = "lc_messages";
  SessionVariable["LcMessagesDir"] = "lc_messages_dir";
  SessionVariable["LcTimeNames"] = "lc_time_names";
  SessionVariable["License"] = "license";
  SessionVariable["LocalInfile"] = "local_infile";
  SessionVariable["LockWaitTimeout"] = "lock_wait_timeout";
  SessionVariable["LockedInMemory"] = "locked_in_memory";
  SessionVariable["LogBin"] = "log_bin";
  SessionVariable["LogBinBasename"] = "log_bin_basename";
  SessionVariable["LogBinIndex"] = "log_bin_index";
  SessionVariable["LogBinTrustFunctionCreators"] = "log_bin_trust_function_creators";
  SessionVariable["LogError"] = "log_error";
  SessionVariable["LogOutput"] = "log_output";
  SessionVariable["LogQueriesNotUsingIndexes"] = "log_queries_not_using_indexes";
  SessionVariable["LogSlaveUpdates"] = "log_slave_updates";
  SessionVariable["LogSlowAdminStatements"] = "log_slow_admin_statements";
  SessionVariable["LogSlowFilter"] = "log_slow_filter";
  SessionVariable["LogSlowRateLimit"] = "log_slow_rate_limit";
  SessionVariable["LogSlowSlaveStatements"] = "log_slow_slave_statements";
  SessionVariable["LogSlowVerbosity"] = "log_slow_verbosity";
  SessionVariable["LogTcSize"] = "log_tc_size";
  SessionVariable["LogWarnings"] = "log_warnings";
  SessionVariable["LongQueryTime"] = "long_query_time";
  SessionVariable["LowPriorityUpdates"] = "low_priority_updates";
  SessionVariable["LowerCaseFileSystem"] = "lower_case_file_system";
  SessionVariable["LowerCaseTableNames"] = "lower_case_table_names";
  SessionVariable["MasterVerifyChecksum"] = "master_verify_checksum";
  SessionVariable["MaxAllowedPacket"] = "max_allowed_packet";
  SessionVariable["MaxBinlogCacheSize"] = "max_binlog_cache_size";
  SessionVariable["MaxBinlogSize"] = "max_binlog_size";
  SessionVariable["MaxBinlogStmtCacheSize"] = "max_binlog_stmt_cache_size";
  SessionVariable["MaxConnectErrors"] = "max_connect_errors";
  SessionVariable["MaxConnections"] = "max_connections";
  SessionVariable["MaxDelayedThreads"] = "max_delayed_threads";
  SessionVariable["MaxDigestLength"] = "max_digest_length";
  SessionVariable["MaxErrorCount"] = "max_error_count";
  SessionVariable["MaxHeapTableSize"] = "max_heap_table_size";
  SessionVariable["MaxInsertDelayedThreads"] = "max_insert_delayed_threads";
  SessionVariable["MaxJoinSize"] = "max_join_size";
  SessionVariable["MaxLengthForSortData"] = "max_length_for_sort_data";
  SessionVariable["MaxLongDataSize"] = "max_long_data_size";
  SessionVariable["MaxPreparedStmtCount"] = "max_prepared_stmt_count";
  SessionVariable["MaxRelayLogSize"] = "max_relay_log_size";
  SessionVariable["MaxSeeksForKey"] = "max_seeks_for_key";
  SessionVariable["MaxSortLength"] = "max_sort_length";
  SessionVariable["MaxSpRecursionDepth"] = "max_sp_recursion_depth";
  SessionVariable["MaxStatementTime"] = "max_statement_time";
  SessionVariable["MaxTmpTables"] = "max_tmp_tables";
  SessionVariable["MaxUserConnections"] = "max_user_connections";
  SessionVariable["MaxWriteLockCount"] = "max_write_lock_count";
  SessionVariable["MetadataLocksCacheSize"] = "metadata_locks_cache_size";
  SessionVariable["MetadataLocksHashInstances"] = "metadata_locks_hash_instances";
  SessionVariable["MinExaminedRowLimit"] = "min_examined_row_limit";
  SessionVariable["MrrBufferSize"] = "mrr_buffer_size";
  SessionVariable["MultiRangeCount"] = "multi_range_count";
  SessionVariable["MyisamBlockSize"] = "myisam_block_size";
  SessionVariable["MyisamDataPointerSize"] = "myisam_data_pointer_size";
  SessionVariable["MyisamMaxSortFileSize"] = "myisam_max_sort_file_size";
  SessionVariable["MyisamMmapSize"] = "myisam_mmap_size";
  SessionVariable["MyisamRecoverOptions"] = "myisam_recover_options";
  SessionVariable["MyisamRepairThreads"] = "myisam_repair_threads";
  SessionVariable["MyisamSortBufferSize"] = "myisam_sort_buffer_size";
  SessionVariable["MyisamStatsMethod"] = "myisam_stats_method";
  SessionVariable["MyisamUseMmap"] = "myisam_use_mmap";
  SessionVariable["Mysql56TemporalFormat"] = "mysql56_temporal_format";
  SessionVariable["NetBufferLength"] = "net_buffer_length";
  SessionVariable["NetReadTimeout"] = "net_read_timeout";
  SessionVariable["NetRetryCount"] = "net_retry_count";
  SessionVariable["NetWriteTimeout"] = "net_write_timeout";
  SessionVariable["Old"] = "old";
  SessionVariable["OldAlterTable"] = "old_alter_table";
  SessionVariable["OldMode"] = "old_mode";
  SessionVariable["OldPasswords"] = "old_passwords";
  SessionVariable["OpenFilesLimit"] = "open_files_limit";
  SessionVariable["OptimizerPruneLevel"] = "optimizer_prune_level";
  SessionVariable["OptimizerSearchDepth"] = "optimizer_search_depth";
  SessionVariable["OptimizerSelectivitySamplingLimit"] = "optimizer_selectivity_sampling_limit";
  SessionVariable["OptimizerSwitch"] = "optimizer_switch";
  SessionVariable["OptimizerUseConditionSelectivity"] = "optimizer_use_condition_selectivity";
  SessionVariable["PerformanceSchema"] = "performance_schema";
  SessionVariable["PerformanceSchemaAccountsSize"] = "performance_schema_accounts_size";
  SessionVariable["PerformanceSchemaDigestsSize"] = "performance_schema_digests_size";
  SessionVariable["PerformanceSchemaEventsStagesHistoryLongSize"] = "performance_schema_events_stages_history_long_size";
  SessionVariable["PerformanceSchemaEventsStagesHistorySize"] = "performance_schema_events_stages_history_size";
  SessionVariable["PerformanceSchemaEventsStatementsHistoryLongSize"] = "performance_schema_events_statements_history_long_size";
  SessionVariable["PerformanceSchemaEventsStatementsHistorySize"] = "performance_schema_events_statements_history_size";
  SessionVariable["PerformanceSchemaEventsWaitsHistoryLongSize"] = "performance_schema_events_waits_history_long_size";
  SessionVariable["PerformanceSchemaEventsWaitsHistorySize"] = "performance_schema_events_waits_history_size";
  SessionVariable["PerformanceSchemaHostsSize"] = "performance_schema_hosts_size";
  SessionVariable["PerformanceSchemaMaxCondClasses"] = "performance_schema_max_cond_classes";
  SessionVariable["PerformanceSchemaMaxCondInstances"] = "performance_schema_max_cond_instances";
  SessionVariable["PerformanceSchemaMaxDigestLength"] = "performance_schema_max_digest_length";
  SessionVariable["PerformanceSchemaMaxFileClasses"] = "performance_schema_max_file_classes";
  SessionVariable["PerformanceSchemaMaxFileHandles"] = "performance_schema_max_file_handles";
  SessionVariable["PerformanceSchemaMaxFileInstances"] = "performance_schema_max_file_instances";
  SessionVariable["PerformanceSchemaMaxMutexClasses"] = "performance_schema_max_mutex_classes";
  SessionVariable["PerformanceSchemaMaxMutexInstances"] = "performance_schema_max_mutex_instances";
  SessionVariable["PerformanceSchemaMaxRwlockClasses"] = "performance_schema_max_rwlock_classes";
  SessionVariable["PerformanceSchemaMaxRwlockInstances"] = "performance_schema_max_rwlock_instances";
  SessionVariable["PerformanceSchemaMaxSocketClasses"] = "performance_schema_max_socket_classes";
  SessionVariable["PerformanceSchemaMaxSocketInstances"] = "performance_schema_max_socket_instances";
  SessionVariable["PerformanceSchemaMaxStageClasses"] = "performance_schema_max_stage_classes";
  SessionVariable["PerformanceSchemaMaxStatementClasses"] = "performance_schema_max_statement_classes";
  SessionVariable["PerformanceSchemaMaxTableHandles"] = "performance_schema_max_table_handles";
  SessionVariable["PerformanceSchemaMaxTableInstances"] = "performance_schema_max_table_instances";
  SessionVariable["PerformanceSchemaMaxThreadClasses"] = "performance_schema_max_thread_classes";
  SessionVariable["PerformanceSchemaMaxThreadInstances"] = "performance_schema_max_thread_instances";
  SessionVariable["PerformanceSchemaSessionConnectAttrsSize"] = "performance_schema_session_connect_attrs_size";
  SessionVariable["PerformanceSchemaSetupActorsSize"] = "performance_schema_setup_actors_size";
  SessionVariable["PerformanceSchemaSetupObjectsSize"] = "performance_schema_setup_objects_size";
  SessionVariable["PerformanceSchemaUsersSize"] = "performance_schema_users_size";
  SessionVariable["PidFile"] = "pid_file";
  SessionVariable["PluginDir"] = "plugin_dir";
  SessionVariable["PluginMaturity"] = "plugin_maturity";
  SessionVariable["Port"] = "port";
  SessionVariable["PreloadBufferSize"] = "preload_buffer_size";
  SessionVariable["Profiling"] = "profiling";
  SessionVariable["ProfilingHistorySize"] = "profiling_history_size";
  SessionVariable["ProgressReportTime"] = "progress_report_time";
  SessionVariable["ProtocolVersion"] = "protocol_version";
  SessionVariable["ProxyUser"] = "proxy_user";
  SessionVariable["PseudoSlaveMode"] = "pseudo_slave_mode";
  SessionVariable["PseudoThreadId"] = "pseudo_thread_id";
  SessionVariable["QueryAllocBlockSize"] = "query_alloc_block_size";
  SessionVariable["QueryCacheLimit"] = "query_cache_limit";
  SessionVariable["QueryCacheMinResUnit"] = "query_cache_min_res_unit";
  SessionVariable["QueryCacheSize"] = "query_cache_size";
  SessionVariable["QueryCacheStripComments"] = "query_cache_strip_comments";
  SessionVariable["QueryCacheType"] = "query_cache_type";
  SessionVariable["QueryCacheWlockInvalidate"] = "query_cache_wlock_invalidate";
  SessionVariable["QueryPreallocSize"] = "query_prealloc_size";
  SessionVariable["RandSeed1"] = "rand_seed1";
  SessionVariable["RandSeed2"] = "rand_seed2";
  SessionVariable["RangeAllocBlockSize"] = "range_alloc_block_size";
  SessionVariable["ReadBufferSize"] = "read_buffer_size";
  SessionVariable["ReadOnly"] = "read_only";
  SessionVariable["ReadRndBufferSize"] = "read_rnd_buffer_size";
  SessionVariable["RelayLog"] = "relay_log";
  SessionVariable["RelayLogBasename"] = "relay_log_basename";
  SessionVariable["RelayLogIndex"] = "relay_log_index";
  SessionVariable["RelayLogInfoFile"] = "relay_log_info_file";
  SessionVariable["RelayLogPurge"] = "relay_log_purge";
  SessionVariable["RelayLogRecovery"] = "relay_log_recovery";
  SessionVariable["RelayLogSpaceLimit"] = "relay_log_space_limit";
  SessionVariable["ReplicateAnnotateRowEvents"] = "replicate_annotate_row_events";
  SessionVariable["ReplicateDoDb"] = "replicate_do_db";
  SessionVariable["ReplicateDoTable"] = "replicate_do_table";
  SessionVariable["ReplicateEventsMarkedForSkip"] = "replicate_events_marked_for_skip";
  SessionVariable["ReplicateIgnoreDb"] = "replicate_ignore_db";
  SessionVariable["ReplicateIgnoreTable"] = "replicate_ignore_table";
  SessionVariable["ReplicateWildDoTable"] = "replicate_wild_do_table";
  SessionVariable["ReplicateWildIgnoreTable"] = "replicate_wild_ignore_table";
  SessionVariable["ReportHost"] = "report_host";
  SessionVariable["ReportPassword"] = "report_password";
  SessionVariable["ReportPort"] = "report_port";
  SessionVariable["ReportUser"] = "report_user";
  SessionVariable["RowidMergeBuffSize"] = "rowid_merge_buff_size";
  SessionVariable["SecureAuth"] = "secure_auth";
  SessionVariable["SecureFilePriv"] = "secure_file_priv";
  SessionVariable["ServerId"] = "server_id";
  SessionVariable["SkipExternalLocking"] = "skip_external_locking";
  SessionVariable["SkipNameResolve"] = "skip_name_resolve";
  SessionVariable["SkipNetworking"] = "skip_networking";
  SessionVariable["SkipParallelReplication"] = "skip_parallel_replication";
  SessionVariable["SkipReplication"] = "skip_replication";
  SessionVariable["SkipShowDatabase"] = "skip_show_database";
  SessionVariable["SlaveCompressedProtocol"] = "slave_compressed_protocol";
  SessionVariable["SlaveDdlExecMode"] = "slave_ddl_exec_mode";
  SessionVariable["SlaveDomainParallelThreads"] = "slave_domain_parallel_threads";
  SessionVariable["SlaveExecMode"] = "slave_exec_mode";
  SessionVariable["SlaveLoadTmpdir"] = "slave_load_tmpdir";
  SessionVariable["SlaveMaxAllowedPacket"] = "slave_max_allowed_packet";
  SessionVariable["SlaveNetTimeout"] = "slave_net_timeout";
  SessionVariable["SlaveParallelMaxQueued"] = "slave_parallel_max_queued";
  SessionVariable["SlaveParallelMode"] = "slave_parallel_mode";
  SessionVariable["SlaveParallelThreads"] = "slave_parallel_threads";
  SessionVariable["SlaveRunTriggersForRbr"] = "slave_run_triggers_for_rbr";
  SessionVariable["SlaveSkipErrors"] = "slave_skip_errors";
  SessionVariable["SlaveSqlVerifyChecksum"] = "slave_sql_verify_checksum";
  SessionVariable["SlaveTransactionRetries"] = "slave_transaction_retries";
  SessionVariable["SlaveTypeConversions"] = "slave_type_conversions";
  SessionVariable["SlowLaunchTime"] = "slow_launch_time";
  SessionVariable["WsrepMaxWsRows"] = "wsrep_max_ws_rows";
  SessionVariable["WsrepMaxWsSize"] = "wsrep_max_ws_size";
  SessionVariable["WsrepMysqlReplicationBundle"] = "wsrep_mysql_replication_bundle";
  SessionVariable["WsrepNodeAddress"] = "wsrep_node_address";
  SessionVariable["WsrepNodeIncomingAddress"] = "wsrep_node_incoming_address";
  SessionVariable["WsrepNodeName"] = "wsrep_node_name";
  SessionVariable["WsrepNotifyCmd"] = "wsrep_notify_cmd";
  SessionVariable["WsrepOn"] = "wsrep_on";
  SessionVariable["WsrepPatchVersion"] = "wsrep_patch_version";
  SessionVariable["WsrepProvider"] = "wsrep_provider";
  SessionVariable["WsrepProviderOptions"] = "wsrep_provider_options";
  SessionVariable["WsrepRecover"] = "wsrep_recover";
  SessionVariable["WsrepReplicateMyisam"] = "wsrep_replicate_myisam";
  SessionVariable["WsrepRestartSlave"] = "wsrep_restart_slave";
  SessionVariable["WsrepRetryAutocommit"] = "wsrep_retry_autocommit";
  SessionVariable["WsrepSlaveFkChecks"] = "wsrep_slave_fk_checks";
  SessionVariable["WsrepSlaveUkChecks"] = "wsrep_slave_uk_checks";
  SessionVariable["WsrepSlaveThreads"] = "wsrep_slave_threads";
  SessionVariable["WsrepSstAuth"] = "wsrep_sst_auth";
  SessionVariable["WsrepSstDonor"] = "wsrep_sst_donor";
  SessionVariable["WsrepSstDonorRejectsQueries"] = "wsrep_sst_donor_rejects_queries";
  SessionVariable["WsrepSstMethod"] = "wsrep_sst_method";
  SessionVariable["WsrepSstReceiveAddress"] = "wsrep_sst_receive_address";
  SessionVariable["WsrepStartPosition"] = "wsrep_start_position";
  SessionVariable["WsrepSyncWait"] = "wsrep_sync_wait";
})(SessionVariable || (SessionVariable = {}));

var _default = SessionVariable;
exports.default = _default;
